source/libs/page-detect.ts

```diff
@@ -77,6 +77,8 @@ export const isRepo = (): boolean => /^[^/]+\/[^/]+/.test(getCleanPathname()) &&
 	!isGist() &&
 	!isRepoSearch();
 
+export const isOwnRepo = (): boolean => isRepo() && getUsername() === getOwnerAndRepo().ownerName;
```

Comment:
I think this can be tested in JSdom

Reply:
getUsername depends on DOM

Reply:
It's basically a combination of two base functions. I don't see any jsdom tests to learn from.

Reply:
Indeed. The `is404` test in `tests/page-detect` technically also depends on the `<title>` tag, but it's more easily-accessible via `document.title`. Perhaps something like:

```js
test('isOwnRepo', t => {
	document.head.insertAdjacentHTML('<meta name="user-login" content="sindresorhus">');
	location.href = 'https://github.com/sindresorhus/refined-github/pull/1929'
	t.true(pageDetect.isOwnRepo());
});
```

The problem is that `isOwnRepo` is a one-time function, so unless @sindresorhus suggests a way to test it further, we can only do this once

Reply:
Actually this function is incorrect and should be dropped. This is what was in `remove-projects-tab`

```js
	// If there's a settings tab, the current user can disable the projects,
	// so the tab should not be hidden
	if (select.exists([
		'.js-repo-nav [data-selected-links^="repo_settings"]', // In repos
		'.pagehead-tabs-item[href$="/settings/profile"]' // In organizations
	].join())) {
		return;
	}
```

Which is different from `currentUser === repoOwner`. This code should also work in organizations where the owner doesn't match the `repoOwner` part of the URL, like https://github.com/xojs/xo

Reply:
basically, copy and pasted the selectors. I assume it applies exactly in the same way.